website/institutions/model.py

```diff
@@ -128,7 +129,7 @@ def find_one(cls, query=None, deleted=False, **kwargs):
     def load(cls, key):
         from website.models import Node
         try:
-            node = Node.find_one(Q('institution_id', 'eq', key), allow_institution=True)
+            node = Node.find_one(Q('institution_id', 'eq', key) & Q('is_deleted', 'ne', True), allow_institution=True)
```

Comment:
do we want to prevent loading the institution model by id? seems like we should be filtering on view/query right?